webpack.config.cjs

```diff
@@ -5,7 +5,7 @@ module.exports = {
   mode: 'production',
   output: {
     path: path.resolve(__dirname, 'public'),
-    filename: 'bundle.js',
+    filename: '[contenthash].bundle.js',
```

Comment:
Nope. `index.html` has a hardcoded `bundle.js` path and IMHO it is not the best practice to introduce dynamic `index.html` generation.

Reply:
You’re right. I missed it